### PR TITLE
Add possibility to choose data layer

### PR DESCRIPTION
**Pull request by mikozet**

### CODE REVIEW

File: test/helpers/core.js [hunk 1]
```diff
@@ -11,5 +11,6 @@ export default function createCore() {
     relayServiceEndpoint: process.env.RELAY_SERVICE_ENDPOINT,
     safeMasterAddress: process.env.SAFE_ADDRESS,
     subgraphName: process.env.SUBGRAPH_NAME,
+    databaseSource: process.env.DATABASE_SOURCE,
```
Comment (JacqueGM) [inline test/helpers/core.js:14]: can you upload the env file with this already updated? Also is the idea to have only one source? I thought we were thinking on having 2/3 option one as fallback?

Comment (llunaCreixent) [reply test/helpers/core.js:14]: The idea is configure the source option outside the core, and the core only reads the env var. The alternatives are 2 so far and will have a code like:
- `graph-node`
- `land-db`

Comment (llunaCreixent) [reply test/helpers/core.js:14]: btw, the updated env file is: `[.env.example](https://github.com/CirclesUBI/circles-core/pull/105/files#diff-a3046da0d15a27e89f2afe639b25748a7ad4d9290af3e7b1b6c1a5533c8f0a8c)`

**Comment by llunaCreixent** [comment]
Test fail because of the changes introduced in the `subgraph v.1.2.1` scripts: https://github.com/CirclesUBI/circles-subgraph/compare/v1.2.0..v1.2.1#diff-b23204e92f793e233cf28605758e6f1d4849830577492c85f00593f977527e27

So I set up the `subgraph v1.2.0` in the docker repo.

**Comment by llunaCreixent** [comment]
we can work with the issue https://github.com/CirclesUBI/circles-core/issues/107

**Review by llunaCreixent — APPROVED** [review]
Great! thank you @mikozet 
an optimization for this code could be having the functions also as a parameter inside the `src/utils.js > requestIndexedDB()` function.

**Comment by llunaCreixent** [comment]
We also have to test the npm package in the api.

**Pull request merged**